public/js/whiteboard.js

```diff
@@ -1,137 +1,233 @@
 /* global moment, io, SunCalc */
 
-const longitude = -73.675770;
-const latitude = 42.729270;
+const longitude = -73.67577;
+const latitude = 42.72927;
```

Comment:
These should get pulled out into environment variables

Reply:
How would one do this on the client side?

Reply:
Ermmm you wouldn't. My mistake. This should probably be pulled from the server via API which will have it stored in an env var. Alternately, we could use the [HTML Geo API](https://www.w3schools.com/html/html5_geolocation.asp), but this is probably less desirable.

Reply:
https://github.com/techinems/headsup/pull/71